.github/dependabot.yml

```diff
@@ -10,3 +10,13 @@ updates:
     schedule:
       interval: "daily"
     open-pull-requests-limit: 5
+  - package-ecosystem: "docker"
```

Comment:
what value are we expecting from dependabot? In the past it was just too noisy. I was considering removing it altogether...

Reply:
> In the past it was just too noisy. I was considering removing it altogether...

Ok, sure. I can revert the changes then